nemoguardrails/rails/llm/config.py

```diff
@@ -772,8 +774,13 @@ class RailsConfig(BaseModel):
         description="Whether this configuration should use streaming mode or not.",
     )
 
-    passthrough: Optional[bool] = Field(
-        default=None,
+    allow_exceptions: bool = Field(
+        default=False,
+        description="If true, allows the rails to throw/pass exceptions to the user.",
+    )
+
+    passthrough: bool = Field(
+        default=False,
```

Comment:
If I remember correctly, the default `None` value here was important. Otherwise, when you need to join two configs, you can't know if a config explicitly set passthrough to False, or it didn't set anything.